### PR TITLE
Hw5 task3 solution

### PR DESCRIPTION
**Pull request by tsehelnyk**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +17,28 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> line = new ArrayList<String>();
+        try {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try with resources?

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +17,28 @@
...
+            File file = new File(fileName);
+            FileReader fileReader = new FileReader(file);
+            BufferedReader reader = new BufferedReader(fileReader);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]:
```suggestion
            BufferedReader bufferedReader = new BufferedReader(fileReader);
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -9,6 +17,28 @@
...
+            String readedLine;
+            while ((readedLine = reader.readLine()) != null) {
+                String[] convertedLine = readedLine.toLowerCase().replaceAll("[^a-zA-Z,\" \"]+", "")
+                        .split(" ");
+                for (String word : convertedLine) {
+                    if (word.startsWith("w")) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: make "w" constant of a class

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -9,6 +17,28 @@
...
+                        line.add(word);
+                    }
+                }
+            }
+            if (line.isEmpty()) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: can we create a solution without this check?

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (tsehelnyk) [reply src/main/java/core/basesyntax/FileWork.java]: ok

File: src/main/java/core/basesyntax/FileWork.java (hunk 3 above)
Comment (tsehelnyk) [reply src/main/java/core/basesyntax/FileWork.java]: ok

File: src/main/java/core/basesyntax/FileWork.java (hunk 4 above)
Comment (tsehelnyk) [reply src/main/java/core/basesyntax/FileWork.java]: ok

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (tsehelnyk) [reply src/main/java/core/basesyntax/FileWork.java]: What for? 
I can't understand.

Comment (tsehelnyk) [reply src/main/java/core/basesyntax/FileWork.java]: I understood and fixed)

**Pull request closed without merging**